active/NumberBox/NumberBox.md

```diff
@@ -0,0 +1,320 @@
+
+## Description
+Represents a control that can be used to display and edit numbers. This supports validation, increment stepping, and computing inline calculations of basic equations, such as multiplication, division, addition, and subtraction.
+
+## Is this the right control? 
+
+You can use a NumberBox control to capture and display mathematic input. If you need an editable text box that accepts more than numbers, use the [TextBox](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.TextBox) control. If you need an editable text box that accepts passwords or other sensitive input, see [PasswordBox](https://docs.microsoft.com/uwp/api/windows.ui.xaml.controls.passwordbox). If you need a text box to enter search terms, see [AutoSuggestBox](https://docs.microsoft.com/uwp/api/windows.ui.xaml.controls.autosuggestbox). If you need to enter or edit formatted text, see [RichEditBox](https://docs.microsoft.com/uwp/api/windows.ui.xaml.controls.richeditbox).
+
+## Examples
+
+### Create a simple NumberBox
+
+Here's the XAML for a basic NumberBox that demonstrates the default look. Use [x:Bind](https://docs.microsoft.com/en-us/windows/uwp/xaml-platform/x-bind-markup-extension#property-path) to ensure the data displayed to the user remains in sync with the data stored in your app. 
+
+
+```XAML
+<NumberBox Value="{x:Bind Path=ViewModel.NumberBoxValue, Mode=TwoWay}" />
+```
+![An in-focus input field showing 0.](images/numberbox-basic.PNG)
+
+### Labeling NumberBox
+
+Use `Header` or `PlaceholderText` if the purpose of the NumberBox isn't clear. `Header` is visible whether or not the NumberBox has a value. 
+
+```XAML
+<NumberBox Header="Enter expression:"
+    Value="{x:Bind Path=ViewModel.NumberBoxValue, Mode=TwoWay}" />
+```
+
+![A header reading "Enter expression:" above a NumberBox.](images/numberbox-header.PNG)
+
+`PlaceholderText` is displayed inside the NumberBox and disappears once a value has been entered.
+
+```XAML
+<NumberBox PlaceholderText="A + B"
+    Value="{x:Bind Path=ViewModel.NumberBoxValue, Mode=TwoWay}" />
+```
+
+![A NumberBox containing placeholder text reading "A + B".](images/numberbox-placeholder-text.PNG)
+
+### Enable calculation support
+
+Setting the `AcceptsCalculation` property to true enables NumberBox to evaluate basic inline expressions such as multiplication, division, addition, and subtraction using standard order of operations. Evaluation is triggered on loss of focus or when the user presses the "Enter" key. Once an expression is evaluated, the original form of the expression is not preserved.
+
+XAML
+```XAML
+<NumberBox Value="{x:Bind Path=ViewModel.NumberBoxValue, Mode=TwoWay}"
+    AcceptsCalculation="True" />
```

Comment:
Somewhere this needs to document the expression and mechanism used to compute the value. Is it RPN? Infix? What mathmatical operations does it know?

Reply:
Recommend: Yes, please document.

Reply:
Resolved with [this](https://github.com/microsoft/microsoft-ui-xaml-specs/commit/ae21bb2da9c4bb83fed7165c86c32ff035bea2f5) commit.